Added introduction file

nikeshbhawsar opened the pull request:
Review it so that we can merge this back to dojo_rules/master.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Commit:
Added introduction file

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1 @@
+Hi I am an Android developer.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

nikeshbhawsar replied:
added code school path.

Commit:
added preffered path

nikeshbhawsar committed:
Update introduction.md

added skills

nikeshbhawsar committed:
Update introduction.md

nikeshbhawsar committed:
Update introduction.md

Commit:
update

Commit:
update

Commit:
updated features

nikeshbhawsar committed:
Merge pull request #1 from nikeshbhawsar/feature_branch_1

updated features

Commit:
check conflicts

Commit:
check conflicts

Commit:
conflict resolved

Commit:
Merge branch 'feature_branch_1'

Commit:
added another feature

Commit:
Merge branch 'master' into feature_branch_1

Commit:
Merge branch 'feature_branch_1'

Commit:
Changes

nikeshbhawsar committed:
Merge pull request #4 from nikeshbhawsar/feature_branch_1

Changes

Commit:
reverting the wrong commit
changes
pull request #4 from nikeshbhawsar/feature_branch_1"

This reverts commit ed252c5fa1f7a6323c9fbed919097fbe088ab13b, reversing
changes made to 47a2d1c2f23446e63497d8ae768ceaca63e87ac9.

Commit:
revert
changes
pull request #4 from nikeshbhawsar/feature_branch_1"

This reverts commit ed252c5fa1f7a6323c9fbed919097fbe088ab13b, reversing
changes made to 47a2d1c2f23446e63497d8ae768ceaca63e87ac9.

Commit:
Merge branch 'master' of https://github.com/nikeshbhawsar/dojo_rules

Commit:
added skill

Commit:
added skill

nikeshbhawsar committed:
Update introduction.md

codeschool-kiddo committed:
Added new rule about coffee

Commit:
corrected typo and case

nikeshbhawsar committed:
Merge pull request #6 from nikeshbhawsar/new_rules

Too many people are spilling coffee

nikeshbhawsar committed:
Merge pull request #5 from nikeshbhawsar/deadly_skills

Deadly skills

Commit:
typo fixed

Commit:
Merge branch 'new_rules' of https://github.com/nikeshbhawsar/dojo_rules into new_rules

Commit:
typo fixed

nikeshbhawsar committed:
Merge pull request #7 from nikeshbhawsar/new_rules

New rules

Commit:
merge

Commit:
Replaced assassin names

nikeshbhawsar committed:
Merge pull request #8 from nikeshbhawsar/release_branch_1.0

Release branch 1.0
